pincer/core/http.py

```diff
@@ -63,7 +65,7 @@ def __init__(self, token: str, *, version: int = None, ttl: int = 5):
 
         version:
             The discord API version.
-            See `<https://discord.com/developers/docs/reference#api-versioning>`_.
```

Comment:
Wouldn’t this break the link?

Reply:
Gotta check that. I forgot I had to.

Reply:
fixed that. Looks like it wasn't showing in the docs at all before :sweat_smile: